enterprise_access/apps/subsidy_access_policy/subsidy_api.py

```diff
@@ -30,6 +30,38 @@ class TransactionPolicyMismatchError(Exception):
     """
 
 
+def subsidy_learner_aggregate_data_cache_key(subsidy_uuid, policy_uuid=None):
+    return versioned_cache_key('get_subsidy_learners_aggregate_data', subsidy_uuid, policy_uuid)
+
+
+def get_and_cache_subsidy_learners_aggregate_data(subsidy_uuid, policy_uuid=None):
+    """
+    Get aggregated learner data for a given subsidy. This can be optionally further filtered
+    """
+    cache_key = subsidy_learner_aggregate_data_cache_key(subsidy_uuid, policy_uuid)
+    cached_response = request_cache(namespace=REQUEST_CACHE_NAMESPACE).get_cached_response(cache_key)
```

Comment:
The request cache will only cache results for the lifetime of a single request - you'll want to use tiered cache to store this data via memcached for some amount of time, primarily to help performance when multiple pages of members data exists.

Reply:
I didn't realize this was only relevant to the lifetime of the request- nice catch